acceptance-tests/src/test/scala/ch/renku/acceptancetests/MergeRequestsAndIssuesSpec.scala

```diff
@@ -38,7 +38,11 @@ class MergeRequestsAndIssuesSpec
 
     `navigate to the issues tab`
 
-    `verify that the GitLab issues iFrame is visible`
+    // Availability of the iFrame depends on the specifc deployment
+    try `verify that the GitLab issues iFrame is visible`
```

Comment:
This works but if we wanted to go a bit more scala like a better solution would be 
```
Try(`verify that the GitLab issues iFrame is visible`).fold (
      _ =>  `verify that the GitLab issues iFrame cannot be shown`, // the error path
      identity // the successful path
    )
```